.github/actions/install-cocoapods/action.yml

```diff
@@ -41,5 +36,5 @@ runs:
         cd sample
         bundle install
         cd ios
-        NO_FLIPPER=1 bundle exec pod install --no-repo-update
+        NO_FLIPPER=1 bundle exec pod install --deployment --repo-update
```

Comment:
The logic here is this: Use `--repo-update` to update the local podspecs for the dependencies, but use `--deployment` to ensure that the `Podfile/Podfile.lock` files do not change